Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -53,6 +53,11 @@ protected CacheBase()
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether context should be maintained until type has been instantiated or not.
+        /// </summary>
+        public bool MaintainContext { get; set; }
```

Comment:
Also, now I think about it, does this need to be a public property? Your use case of it has it being set when a cache is created. Would a constructor param not be sufficient for this?